### ISSUE TITLE
No update found?

### ISSUE THREAD

**Issue by HatredBlaze** [awgil/ffxiv_bossmod#144]
Sorry if this comes out as stoopid, but I'm new to Github, and I saw that you updated this plugin for EX6 & Zurvan, but it seems like I can't update it in-game ?
Can I abuse a bit of your time and either being told how to update, or if it's just me missing some information maybe ?
Thanks !

**Comment by Nightstalks** [comment]
i too have this issue. maybe the module isnt live?

**Comment by awgil** [comment]
Yeah, I finished ex6 late last night and didn't release a new version, expecting client-side patch today. Since there was none, I'll just release new version later today - that should have ex6.

Zurvan should already be available.

**Comment by awgil** [comment]
Ex6 should work in latest version.
